internal/faustjs.org/docs/guides/custom-post-types.mdx

```diff
@@ -0,0 +1,335 @@
+---
+slug: /guides/custom-post-types
+title: Setting up Custom Post Types (CPTs) in Faust
+description: How to use Faust Templates to display your Custom Post Types
+---
+
+This guide covers how to set up Custom Post Types in your Faust templates.
+
+:::caution
+
+This guide assumes you have a working Faust app using [Faust templates](/docs/templates). If you don't have a Faust site created yet, please follow the [Getting Started](/docs/getting-started) guide first.
+
+:::
+
+## Before You Start
+
+It's important that your CPT is registered with some specific fields and values:
+
+- `public`: Must be `true`. Otherwise the posts can't be resolved in Faust.
+- `has_archive`: Should be `true`. This is so that you can create an archive (i.e. `/movies`) in your Faust app.
+- `show_in_graphql`: Must be `true` so the data can be queried from WPGraphQL.
+
+For example, here is the CPT that was registered for this tutorial:
+
+```php
+function register_movies_cpt() {
+	$args = [
+		"label" => esc_html__( "Movies", "twentytwentyone" ),
+		"public" => true, // If the CPT is not public, it can not be properly resolved in Faust.
+		"publicly_queryable" => true,
+		"show_ui" => true,
+		"show_in_rest" => true,
+		"rest_base" => "",
+		"rest_controller_class" => "WP_REST_Posts_Controller",
+		"rest_namespace" => "wp/v2",
+		"has_archive" => true, // Important for creating an archive in Faust.
+		"show_in_menu" => true,
+		"show_in_nav_menus" => true,
+		"delete_with_user" => false,
+		"exclude_from_search" => false,
+		"capability_type" => "post",
+		"map_meta_cap" => true,
+		"hierarchical" => false,
+		"can_export" => false,
+		"rewrite" => [ "slug" => "movies", "with_front" => true ],
+		"query_var" => true,
+		"supports" => [ "title", "editor", "thumbnail" ],
+		"show_in_graphql" => true,
+		"graphql_single_name" => "Movie",
+		"graphql_plural_name" => "Movies",
+	];
+
+	register_post_type( "movies", $args );
```

Comment:
This isn't up to WordPress coding standards. Replace the " with '

Reply:
This goes for the above array as well.

Reply:
Good call. Addressses in my latest commit 👍